docs/source/platform_support.rst

```diff
@@ -44,9 +44,6 @@ We encourage especially users less experienced with Linux to use one of the supp
 Linux distributions typically ship a default version of Python called "system Python".
 This version of Python can be used with cocotb unless noted otherwise.
 
-* **Red Hat Enterprise Linux (RHEL) 8 amd64**,
-  shipping with Python 3.6 (newer versions of Python are available), pip 9, and glibc 2.28.
-  `Upstream support until May 2029 <https://access.redhat.com/support/policy/updates/errata#Life_Cycle_Dates>`_.
```

Comment:
RHEL is still reasonably widely used in my experience, and also supported by simulator vendors. We can't drop it just yet. We *can*, however, drop Python 3.6, RHEL8 ships up to Python 3.12 in its appstream repos.